app/views/layouts/application.html.erb

```diff
@@ -1,23 +1,32 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>DisneyRideReviewsGroupProject</title>
+  <title>Disney Ride Reviews</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   <%= stylesheet_link_tag 'application', media: 'all' %>
   <%= csrf_meta_tags %>
+  <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
+  <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
+  <meta name="msapplication-TileColor" content="#da532c">
+  <meta name="theme-color" content="#ffffff">
+  <%= favicon_link_tag asset_path('favicon.ico') %>
 </head>
 <body>
   
   <nav class="top-bar">
-    <h1><a href="/">DisneyRideReviewsGroupProject</a></h1>
+    
+    <h1><a href="/rides"><img src="assets/Logo - final.png" alt='logo' class='logo' /></a> </h1>
```

Comment:
we should use the `image_tag` helper
https://api.rubyonrails.org/classes/ActionView/Helpers/AssetTagHelper.html#method-i-image_tag